src/test/test-honor-first-shutdown.service

```diff
@@ -0,0 +1,11 @@
+[Unit]
+Description=Honor First Shutdown feature
+After=multi-user.target
+
+[Service]
+ExecStart=/test-honor-first-shutdown.sh
+ExecStop=/bin/sh -x -c 'kill -SIGKILL $MAINPID'
```

Comment:
The shell is not nedeed here. Just `ExecStop=kill -SIGKILL $MAINPID` .

Reply:
@keszybz I have tried to use this ExecStop without the shell but simply cannot make it work. I found that testsuite-09 uses the same shell to implement a kill in it's ExecStop. When I remove the shell and run the test I get the following error when running the image.

TEST-48-HONORFIRSTSHUTDOWN RUN: testing honor first shutdown
unit_file_build_name_map: normal unit file: /usr/lib/systemd/tests/testdata/units/test-honor-first-shutdown.service
test-honor-first-shutdown.service: Trying to enqueue job test-honor-first-shutdown.service/start/replace
Sent message type=error sender=org.freedesktop.systemd1 destination=n/a path=n/a interface=n/a member=n/a cookie=1 reply_cookie=1 signature=s error-name=org.freedesktop.systemd1.BadUnitSetting error-message=Unit test-honor-first-shutdown.service has a bad unit file setting.
Failed to process message type=method_call sender=n/a destination=org.freedesktop.systemd1 path=/org/freedesktop/systemd1 interface=org.freedesktop.systemd1.Manager member=StartUnit cookie=1 reply_cookie=0 signature=ss error-name=n/a error-message=n/a: Unit test-honor-first-shutdown.service has a bad unit file setting.

I using the shell operation is not acceptable please help me understand how to fix it.